src/xrdok.js

```diff
@@ -681,3 +681,280 @@ const XRiconchevron = 'xr-icon-chevron';
+    planeGraphSecondaryEl.setObject3D('xr-secondary-plane-graph', planeGraphSecondaryMesh);
+
+    planeGraphEl.appendChild(planeGraphPrimaryEl);
+    planeGraphEl.appendChild(planeGraphSecondaryEl);
+
+    planeGraphEl.setAttribute('visible', data.showGraph);
+    return planeGraphEl;
+  }
+
+  function createData(data, el) {
+    // parse CSV, if available
+    if (data.src) {
+      Papa.parse(data.src, {  // eslint-disable-line no-undef
+        download: true,
+        complete: function(res) {
+          data.descriptors = [];
+          data.primaryValues = [];
+          data.secondaryValues = [];
+          for (var i = 0; i < res.data.length; i++) {
+            data.descriptors.push(res.data[i][0]);
+            data.primaryValues.push(parseFloat(res.data[i][1]));
+            data.secondaryValues.push(parseInt(res.data[i][2]));
+          }
+
+          el.appendChild(createGeometryLine(data));
+          el.appendChild(createGeometryPlane(data));
+          el.appendChild(createLegend(data));
+        }
+      });
+    }
+  }
+
+  AFRAME.registerComponent(XRplanegraph, {
+
+    schema: {
+      color: { type: 'color', default: '#ccfccf'},
+      colorMax: { type: 'color', default: '#fccfcc'},
+      colorMin: { type: 'color', default: '#ccfccf'},
+      descriptors: {type: 'array', default: [] },
+      primaryValues: { type: 'array', default: [] },
+      primaryValuesBase: { type: 'number', default: 0},
+      secondaryValues: { type: 'array', default: [] },
+      secondaryValuesBase: { type: 'number', default: 1 },
+      showGraph: { type: 'boolean', default: true},
+      showLegend: { type: 'boolean', default: true},
+      showLine: { type: 'boolean', default: true},
+      src: { type: 'string', default: './models/lufttemperatur_bbb_sortiert.csv'},
+    },
+
+    init: function() {
+      // data and element
+      var data = this.data;
+      var el = this.el;
+
+      createData(data, el);
+
+      this.lineEl = document.getElementById('line-graph-parent');
+      this.graphEl = document.getElementById('plane-graph-parent');
+      this.legendEl = document.getElementById('legend-parent');
+    },
+
+    tick: function() {
+      if (this.legendEl !== null && this.activeCamera) {
+        var legendEl = this.legendEl;
+        var activeCameraPosition = new THREE.Vector3();
+        this.activeCamera.object3D.getWorldPosition(activeCameraPosition);
+        for (let i =0; i < legendEl.children.length; i++){
+          if (legendEl.children[i].tagName.toLowerCase() === 'a-text') {
+            legendEl.children[i].object3D.lookAt(activeCameraPosition);
+          }
+        }
+      } else {
+        this.activeCamera = document.querySelector('[camera]');
+        this.legendEl = document.getElementById('legend-parent');
+      }
+    },
+
+    update: function(oldData) {
+      var data = this.data;
+
+      const lineEl = document.getElementById('line-graph-parent');
```

Comment:
Thats why you have to do the assignments again here. Because the elements are just not there on the first run.